xml/deployment_cifs.xml

```diff
@@ -328,19 +352,40 @@
       </step>
       <step>
        <para>
-        Initialize the cluster on &si.earth;:
+        Map the RBD image <literal>rbd</literal> on both &sgw;s via
```

Comment:
Sorry, I missed this previously - the RBD image name here should also be _sbd01_ . Looks good otherwise.